features/step_definitions/user_signup_steps.rb

```diff
@@ -0,0 +1,11 @@
+Then(/^I should be on the signup page$/) do
+  expect(page.current_path).to eq new_user_registration_path
+end
+
+Given(/^I am on the signup page$/) do
+  visit new_user_registration_path
+end
+
+Then(/^I should be on the signup error page$/) do
```

Comment:
You seem to be using this step a lot, how can you DRY this up to avoid re-implementing the same step every time you test to see if you're on a given page?